PUMI/pipelines/anat/segmentation.py

```diff
@@ -129,9 +129,9 @@ def bet_fsl(wf, fmri=False, volume='middle', **kwargs):
         wf.connect(bet, 'out_file', qc, 'overlay')
         wf.connect('inputspec', 'in_file', qc, 'background')
 
-    # sinking
-    wf.connect(bet, 'out_file', 'sinker', 'out_file')
-    wf.connect(bet, 'mask_file', 'sinker', 'mask_file')
+        # sinking
```

Comment:
why the indentation? no it only runs in the else branch (if fmri is False or None)